gio-subclass/src/auto/action_group.rs

```diff
@@ -0,0 +1,356 @@
+    let klass = &**(gptr as *const *const ClassStruct<T>);
+    let interface_static = klass.get_interface_static(gio_ffi::g_action_group_get_type())
+                                     as *const ActionGroupStatic<T>;
+    let instance = &*(gptr as *const T::InstanceStructType);
+    let imp = instance.get_impl();
+    let imp = (*(*interface_static).imp_static).get_impl(imp);
+    let wrap = from_glib_borrow(gptr);
+    match imp.get_action_parameter_type(&wrap, &String::from_glib_none(action_name)){ Some(t)  => t/*Not checked*/.to_glib_none().0, None => ptr::null()}
+}
+
+unsafe extern "C" fn actiongroup_get_action_state<T: ObjectType>
+(gptr: *mut gio_ffi::GActionGroup, action_name: *const c_char) -> *mut glib_ffi::GVariant
+{
+    callback_guard!();
+    floating_reference_guard!(gptr);
+    let klass = &**(gptr as *const *const ClassStruct<T>);
+    let interface_static = klass.get_interface_static(gio_ffi::g_action_group_get_type())
+                                     as *const ActionGroupStatic<T>;
+    let instance = &*(gptr as *const T::InstanceStructType);
+    let imp = instance.get_impl();
+    let imp = (*(*interface_static).imp_static).get_impl(imp);
+    let wrap = from_glib_borrow(gptr);
+    match imp.get_action_state(&wrap, &String::from_glib_none(action_name)){ Some(t)  => t.to_glib_full(), None => ptr::null_mut()}
+}
+
+unsafe extern "C" fn actiongroup_get_action_state_hint<T: ObjectType>
+(gptr: *mut gio_ffi::GActionGroup, action_name: *const c_char) -> *mut glib_ffi::GVariant
+{
+    callback_guard!();
+    floating_reference_guard!(gptr);
+    let klass = &**(gptr as *const *const ClassStruct<T>);
+    let interface_static = klass.get_interface_static(gio_ffi::g_action_group_get_type())
+                                     as *const ActionGroupStatic<T>;
+    let instance = &*(gptr as *const T::InstanceStructType);
+    let imp = instance.get_impl();
+    let imp = (*(*interface_static).imp_static).get_impl(imp);
+    let wrap = from_glib_borrow(gptr);
+    match imp.get_action_state_hint(&wrap, &String::from_glib_none(action_name)){ Some(t)  => t.to_glib_full(), None => ptr::null_mut()}
+}
+
+unsafe extern "C" fn actiongroup_get_action_state_type<T: ObjectType>
+(gptr: *mut gio_ffi::GActionGroup, action_name: *const c_char) -> *const glib_ffi::GVariantType
+{
+    callback_guard!();
+    floating_reference_guard!(gptr);
+    let klass = &**(gptr as *const *const ClassStruct<T>);
+    let interface_static = klass.get_interface_static(gio_ffi::g_action_group_get_type())
+                                     as *const ActionGroupStatic<T>;
+    let instance = &*(gptr as *const T::InstanceStructType);
+    let imp = instance.get_impl();
+    let imp = (*(*interface_static).imp_static).get_impl(imp);
+    let wrap = from_glib_borrow(gptr);
+    match imp.get_action_state_type(&wrap, &String::from_glib_none(action_name)){ Some(t)  => t/*Not checked*/.to_glib_none().0, None => ptr::null()}
```

Comment:
Is the ` Not checked` happening only for `to_glib_none().0` return values? in that case that makes sense, it's most likely always a bug :)